.circleci/config.yml

```diff
@@ -0,0 +1,136 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:8u171-jdk-node-browsers
```

Comment:
Interestingly, it seems `circleci/openjdk:8u181-node` is 696MB while the `*-8u171-jdk-node-browsers` series is more like 898MB.  Could you use the smaller one?

Reply:
Sure. I'm pretty sure we don't actually use the browsers.